### DISCUSSION TITLE
installation with AD  and  azure blob

### DISCUSSION THREAD

**Discussion by gpointer** [drakkan/sftpgo#439] [Q&A] [answered]
Hi, 
We need a new sftp server. We used Debian with  rssh . That's why I looked at sftpgo. If I want to use AD and Azure Blob storage, do I have to install everything in Windows or is that also possible with the Debian installation?
Thanks and Greetings

**Comment by drakkan** [comment, marked as answer]
Hi,

SFTPGo works on Windows, Linux, macOS, FreeBSD, the OS is a your choice. For AD integration you need to write an external authentication hook, several examples are available:

https://github.com/drakkan/sftpgo/tree/main/examples/ldapauth
https://github.com/drakkan/sftpgo/tree/main/examples/ldapauthserver
https://github.com/drakkan/sftpgo/tree/main/examples/php-activedirectory-http-server

**Reply by gpointer** [reply]
thank you :-)
